README.md

```diff
@@ -1,70 +1,221 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
```

Comment:
Kindly make sure you have added the section `Future Features`, including 1-3 potential features to be added. All the main headers from this [README template](https://github.com/microverseinc/readme-template) should be present in your repo. The only optional section is FAQ.